Assets/Scripts/UI/TwitchComponentHandle.cs

```diff
@@ -601,7 +599,11 @@ public IEnumerator OnMessageReceived(string userNickName, string userColor, stri
 
 		if (_solver != null)
 		{
-		    if (!IsAuthorizedDefuser(userNickName)) return null;
+		    if (!IsAuthorizedDefuser(userNickName))
+		    {
+		        ircConnection.SendMessage(TwitchPlaySettings.data.TwitchPlaysDisabled, userNickName);
```

Comment:
This message is already sent by `IsAuthorizedDefuser`, why are you sending it again?